### PR TITLE
Feature/streamline fee settings

### PR DESCRIPTION
**Pull request by malteish**
changes:
1. set fixed denominator for all fees (10000)
2. extract some more fee calculation logic into internal function

### CODE REVIEW

File: contracts/FeeSettings.sol [hunk 1]
```diff
@@ -317,37 +269,13 @@ contract FeeSettings is
      * @param _fees The fees to check
      */
     function checkFeeLimits(Fees memory _fees) internal pure {
+        require(_fees.tokenFeeNumerator <= MAX_TOKEN_FEE_NUMERATOR, "Token fee must be equal or less 5%");
```
Comment (CJentzsch) [inline contracts/FeeSettings.sol:272]: "or less **than** 5%" -> minor grammar check

File: contracts/FeeSettings.sol [hunk 2]
```diff
@@ -317,37 +269,13 @@ contract FeeSettings is
      * @param _fees The fees to check
```
[the remark this answers is not in the document]
Comment (malteish) [reply contracts/FeeSettings.sol:269]: :star_struck:

[the remark this answers is not in the document]
Comment (malteish) [reply contracts/FeeSettings.sol:253]: removed [0a12b89](https://github.com/corpus-io/tokenize.it-smart-contracts/pull/300/commits/0a12b89a185465a84473fa2c769d80bd41874d3c)

File: contracts/FeeSettings.sol (hunk 1 above)
Comment (malteish) [reply contracts/FeeSettings.sol]: fixed [0a12b89](https://github.com/corpus-io/tokenize.it-smart-contracts/pull/300/commits/0a12b89a185465a84473fa2c769d80bd41874d3c)

**Review by CJentzsch — APPROVED** [review]

**Pull request merged**